src/featureFlags/index.tsx

```diff
@@ -13,7 +13,7 @@ export enum FeatureFlag {
   uniswapXExactOutputEnabled = 'uniswapx_exact_output_enabled',
   multichainUX = 'multichain_ux',
   currencyConversion = 'currency_conversion',
-  fotAdjustedmentsEnabled = 'fot_adjustments_enabled',
+  fotAdjustedmentsEnabled = 'fot_dynamic_adjustments_enabled',
```

Comment:
super nit and non-blocking but this long name is pushing over the dropdown, may need to add truncation
![Screenshot 2023-09-07 at 1 45 43 PM](https://github.com/Uniswap/interface/assets/11512321/8a747e0c-6a74-4ecb-a789-e8e317b8b3ab)